ggml-cuda.cu

```diff
@@ -8057,7 +8057,7 @@ bool ggml_cuda_compute_forward(struct ggml_compute_params * params, struct ggml_
     if (tensor->op == GGML_OP_MUL_MAT) {
         if (tensor->src[0]->ne[3] != tensor->src[1]->ne[3]) {
 #ifndef NDEBUG
-            fprintf(stderr, "%s: cannot compute %s: src0->ne[3] = %d, src1->ne[3] = %d - fallback to CPU\n", __func__, tensor->name, tensor->src[0]->ne[3], tensor->src[1]->ne[3]);
+            fprintf(stderr, "%s: cannot compute %s: src0->ne[3] = %ld, src1->ne[3] = %ld - fallback to CPU\n", __func__, tensor->name, tensor->src[0]->ne[3], tensor->src[1]->ne[3]);
```

Comment:
I believe the portable way to print a 64-bit integer is with `PRId64`.

Reply:
Yes, the best way is to `#include <cinttypes>` and use e.g. `... = " PRId64 ", src1->...`, so we aren't assuming that sizeof(long) == 64, which is false on e.g. 64-bit Windows which is LLP64.

Reply:
@haohui We're still missing the #include.